apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -341,7 +464,8 @@ const CustomizableCurriculumCatalogCard = ({
             </div>
           </div>
         </div>
-        {isAssignDialogOpen && renderAssignDialog()}
+        {isAssignDialogOpen &&
+          createPortal(renderAssignDialog(), document.body)}
```

Comment:
What does `createPortal` do here?

Reply:
It breaks the dialog out of the parent div. It's actually not necessary for this change but it is needed when there are multiple curriculum in a carousel view.